index.html

```diff
@@ -1,11 +1,454 @@
+            </li>
+            <li class="client-icons__item">
+              <a href="" class="client-icons__link"
+                ><svg class="client-icons__svg">
+                  <use href="./images/svg/icons.svg#company_tagline"></use>
+                </svg>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <div class="container">
+        <div class="footer-content">
+          <div class="footer__info">
+            <a href="./index.html" class="logo logo-footer"
+              ><span class="logo__web">Web</span
+              ><span class="logo__studio--white">Studio</span></a
+            >
+            <div class="footer__content">
+              <address class="address">м. Київ, пр-т Лесі Українки, 26</address>
+
+              <div class="footer__contacts contacts">
+                <a
+                  href="mailto:info@devstudio.com"
+                  class="contacts__link-footer"
+                >
+                  info@devstudio.com</a
+                >
+
+                <a href="tel:+380961111111" class="contacts__link-footer"
+                  >+38 096 111 11 11</a
+                >
+              </div>
+            </div>
+          </div>
+          <div class="footer__social social-media--footer">
+            <h4 class="social__title">приєднуйтесь</h4>
+            <ul class="social-icons__list">
+              <li class="social-icons__item">
+                <a
+                  href="#"
+                  class="social-icons__link social-icons__link--footer"
+                  ><svg class="social-icons__svg--footer icon--insta">
+                    <use href="./images/svg/icons.svg#insta"></use>
+                  </svg>
+                </a>
+              </li>
+              <li class="social-icons__item">
+                <a
+                  href="#"
+                  class="social-icons__link social-icons__link--footer"
+                  ><svg class="social-icons__svg--footer icon--twitter">
+                    <use href="./images/svg/icons.svg#twitter"></use>
+                  </svg>
+                </a>
+              </li>
+              <li class="social-icons__item">
+                <a
+                  href="#"
+                  class="social-icons__link social-icons__link--footer"
+                  ><svg class="social-icons__svg--footer icon--facebook">
+                    <use href="./images/svg/icons.svg#facebook"></use>
+                  </svg>
+                </a>
+              </li>
+              <li class="social-icons__item">
+                <a
+                  href="#"
+                  class="social-icons__link social-icons__link--footer"
+                  ><svg class="social-icons__svg--footer icon--linkedin">
+                    <use href="./images/svg/icons.svg#linkedin"></use>
+                  </svg>
+                </a>
+              </li>
+            </ul>
```

Comment:
Here should be the `social-icons` block, with certain modifiers where appropriate